config/react-config.json

```diff
@@ -17,5 +17,13 @@
     "i18nConfigPath": "/bahmni_config/openmrs/i18n/",
     "BAHMNI_CONFIG_URL": "/bahmni_config/openmrs/apps",
     "IMPLEMENTATION_CONFIG_URL": "/implementation_config/openmrs/apps",
-    "locationTagName": "Appointment Location"
+    "locationTagName": "Appointment Location",
+    "otpServiceUrl": "/otp-service",
+    "personUrl": "/openmrs/ws/rest/v1/person/",
+    "messageType": {
+        "appointmentBookingMessage": "APPOINTMENT_BOOKING_MESSAGE",
+        "recurringAppointmentBookingMessage": "RECURRING_APPOINTMENT_BOOKING_MESSAGE",
+        "teleconsultationAppointmentBookingMessage": "TELECONSULTATION_APPOINTMENT_BOOKING_MESSAGE"
+    },
+    "clinicName": "Bahmni"
```

Comment:
This should not be in "code". Should be in a configuration repository. Where is the clinic name read from for Prescription printouts? cc: @gokultw

Reply:
We are fetching the name from default location.
https://github.com/BahmniIndiaDistro/clinic-config/blob/main/openmrs/apps/customDisplayControl/js/customControl.js#L307

Reply:
Refactored to read clinic name from the login location